Expensify/App
# fix cursor issue

### Details

On Android, TextInput padding not working when its parent has display: none/flex to show/hide. 
In this case, position: 'absolute' works fine when the TextInput will be mounted/unmounted.
So,  we are using `width` to toggle visibility because display: 'none' on the parent affects padding styles on the TextInput.

### Fixed Issues

Fixes #987

### Tests

1. Login into the application on Android emulator or devices.
2. Click hamburger menu on Home screen to open sidebar
3. On the left sidebar, You can just see that the cursor doesn't look too close to the left edge of the search box.

### Screenshots

![103078436-7a598f00-4597-11eb-9bf9-4b7c3ef75ccd](https://user-images.githubusercontent.com/74012504/103300002-81333800-49c3-11eb-87ee-a3fb86b46cc2.png)
